package.json

```diff
@@ -16,15 +16,18 @@
     "fifo": "^2.3.0",
     "mic": "^2.1.1",
     "object.pick": "^1.2.0",
-    "pigpio": "^0.4.0",
-    "rpi-ws281x-native": "^0.8.1",
     "semaphore": "^1.0.5",
     "sleep": "^5.0.0",
     "temp": "^0.8.3",
     "twitter": "^1.5.0",
     "web-audio-api": "^0.2.2",
     "watson-developer-cloud": "^2.11.1"
   },
+  "optionalDependencies": {
+    "//": "pigpio and pi-ws281x-native are raspberry exclusive. To the project works in Linux it should be keeped as optional.",
```

Comment:
Yes, we want it for RPi ... 
However, being able to prototype and test fast on a mac/linux is a **huge benefit**. Many dev may not have the bot next to them at any given time or not have the right ip address to deploy and test things. I vote we support limited testing on mac/linux . Ofcourse things like led, waving, camera can only be tested on the pi.